tests/Unit/WhatsAppCloudApiTest.php

```diff
@@ -707,6 +710,75 @@ public function test_send_contact()
         $this->assertEquals(false, $response->isError());
     }
 
+	public function test_send_list()
+	{
+		$to = $this->faker->phoneNumber;
+		$url = $this->buildRequestUri();
+
+		$listHeader = ['type' => 'text', 'text' => $this->faker->text(60)];
+		$listBody = ['text' => $this->faker->text(1024)];
+		$listFooter = ['text' => $this->faker->text(60)];
+
+		$generateListRowId = fn () => strtolower(trim(preg_replace('/[^A-Za-z0-9-]+/', '-', substr($this->faker->text(100), 0, -1))));
```

Comment:
You don't need complex code to create unique ids for testing. You can use `this->faker->uuid` in lines 725 and 726

Reply:
Done! Thanks for the tip.